config/settings.yml

```diff
@@ -16,7 +16,7 @@ kafka_producer_topics:
   notifications: 'platform.notifications.ingress'
 compliance_ssg_url: http://compliance-ssg:8080
 host_inventory_url: http://insights-inventory.platform-ci.svc.cluster.local:8080
-rbac_url: http://rbac.rbac-ci.svc.cluster.local:9002
+rbac_url: http://rbac:9002
```

Comment:
I think this will break the CI and you even should not do it. In production we do have RBAC up and running and it should not be tampered with.